test/e2e/deploy_test.go

```diff
@@ -0,0 +1,126 @@
+package e2e_test
+
+import (
+	"os"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/pborman/uuid"
+
+	cli "github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/utils"
+	. "github.com/onsi/gomega/gbytes"
+	"github.com/onsi/gomega/gexec"
+)
+
+var _ = Describe("Deploy simple cluster", func() {
+
+	It("Release sample all-in-one.yaml should work", func() {
+		By("Prepare namespaces")
```

Comment:
[nit] I found the design proposed by @vasilevp with `By` with code in callbacks looking very nice, WDYT about using it here?